src/schnetpack/task.py

```diff
@@ -204,3 +227,31 @@ def optimizer_step(
 
         # update params
         optimizer.step(closure=optimizer_closure)
+
+
+class ConsiderOnlySelectedAtoms(nn.Module):
+    """
+    Constraint that allows to adapt the use of atom-wise targets such as, e.g., atomic forces for model optimization.
+    In the forward pass, a torch tensor is loaded from the dataset, which specifies the considered atoms is. Only the
```

Comment:
there is something wrong in this sentence. also the docstring could be clearer, perhaps with a usage example